### PR TITLE
Epic 7/story integrate firebase auth with google calendar 78

### PR DESCRIPTION
**Pull request by Weigiu**
Get classroom from google calendar #10

### CODE REVIEW

File: SD-K9/src/app/pages/queued-route/queued-route-store.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    private _ADD: string = 'ADD';
+    private _SHIFT: string = 'SHIFT';
+
+    private _queuedRouteReducer: StateReducer<QueuedRouteState> = (state: QueuedRouteState, action: StateAction): QueuedRouteState => {
+        switch(action.type) {
+            case this._ADD: {
+                state.routes.push(action.payload);
+                return state;
+              }
+              case this._SHIFT: {
```
[the remark this answers is not in the document]
Comment (Weigiu) [reply SD-K9/src/app/pages/queued-route/queued-route-store.ts:27]: removed shift case , only needs clear and add

File: SD-K9/src/app/components/pullup/pullup.component.html [hunk 2]
```diff
@@ -20,64 +16,20 @@
...
-          </div>
-          <div class="near-me-icon-container">
-            <ion-fab-button (click)="showNearMe('park')" color="secondary" class="near-me-icon">
-              <img style="width: 28px" src="../../../assets/images/tree-alt-light.svg" alt="parks near me">
-            </ion-fab-button>
-            <div class="near-me-label">Parks</div>
-          </div>
-        </ion-card-content>
-      </ion-card>
-
```
Comment (david0455) [inline SD-K9/src/app/components/pullup/pullup.component.html:80]: Why did you remove this block of code

Comment (samiha-rahman) [reply SD-K9/src/app/components/pullup/pullup.component.html]: Fixed
